jsapp/js/account/plans/plansRoute.component.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import {UsageContext, useUsage} from 'js/account/usage/useUsage.hook';
+import {ProductsContext, useProducts} from '../useProducts.hook';
+import Plan from './plan.component';
+
+export default function plansRoute() {
+  const usage = useUsage();
+  const products = useProducts();
+
+  return (
+    <UsageContext.Provider value={usage}>
+      <ProductsContext.Provider value={products}>
+        <Plan />
+      </ProductsContext.Provider>
+    </UsageContext.Provider>
```

Comment:
I have gone ahead and moved the context providers to a new `BillingOutlet`. It was a little awkward because the routes aren't nested under their own url path, so they are all index routes now. Let me know what you think.